R/colour-text.r

```diff
@@ -5,9 +8,19 @@ testthat_colours <- list(
   error = crayon::red
 )
 
-colourise <- function(text, as = c("passed", "skipped", "error")) {
+#' @export
+colourise <- function(text, as = c("passed", "skipped", "error"), fg, bg) {
+  fg_warn <- function() warning("The fg parameter to testthat::colourise is deprecated.", call. = FALSE)
+  if (!missing(fg)) fg_warn()
+  if (any(is.element(as, allowed_colors))) {
```

Comment:
It seems so, the old version gives colored results for capital versions of colors.